code/datums/status_effects/drugs.dm

```diff
@@ -0,0 +1,64 @@
+/datum/status_effect/woozy
+	id = "woozy"
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/woozy
+
+
+/datum/status_effect/woozy/nextmove_modifier()
+	return 1.5
+
+/atom/movable/screen/alert/status_effect/woozy
+	name = "Woozy"
+	desc = "You feel a bit slower than usual, it seems doing things with your hands takes longer than it usually does"
+	icon_state = "woozy"
+
+
+/datum/status_effect/high_blood_pressure
+	id = "high_blood_pressure"
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/high_blood_pressure
+
+/datum/status_effect/high_blood_pressure/on_apply()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.physiology.bleed_mod *= 1.25
+
+/datum/status_effect/high_blood_pressure/on_remove()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.physiology.bleed_mod *= 0.80
+
+/atom/movable/screen/alert/status_effect/high_blood_pressure
+	name = "High blood pressure"
+	desc = "Your blood pressure is real high right now ... You'd probably bleed like a stuck pig."
+	icon_state = "highbloodpressure"
+
+
+
+/datum/status_effect/seizure
+	id = "seizure"
+	tick_interval = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/seizure
+
+/datum/status_effect/seizure/on_apply()
+	if(!iscarbon(owner))
+		return FALSE
+	var/amplitude = rand(1 SECONDS, 3 SECONDS)
+	duration = amplitude
+	owner.Jitter(50)
+	owner.Paralyze(duration)
+	owner.visible_message("<span class='warning'>[owner] drops to the ground as [owner.p_they()] start seizing up.</span>",
+	"<span class='warning'>[pick("You can't collect your thoughts...",
+								"You suddenly feel extremely dizzy...",
+								"You cant think straight...",
+								"You can't move your face properly anymore...")]</span>")
+	return TRUE
+
+/atom/movable/screen/alert/status_effect/seizure
+	name = "Seizure"
+	desc = "It would appear you are having a seisure right now"
+	icon_state = "paralysis"
```

Comment:
There really shouldn't be an alert for this, the fact that you seize up is more than enough to tell you this

Reply:
aba9a1c

Got rid of the alert.